monkey/monkey_island/cc/ui/src/components/Main.tsx

```diff
@@ -186,12 +186,21 @@ class AppComponent extends AuthComponent {
     }
   }
 
+  getSideNavHeader() {
+    if(this.state.islandMode === 'ransomware'){
+      return "Ransomware"
+    } else {
+      return "Custom"
+    }
+  }
+
   render() {
 
     let defaultSideNavProps = {completedSteps: this.state.completedSteps,
                                onStatusChange: this.updateStatus,
                                islandMode: this.state.islandMode,
-                               defaultReport: this.getDefaultReport()}
+                               defaultReport: this.getDefaultReport(),
+                               sideNavHeader: this.getSideNavHeader()}
```

Comment:
Instead of `sideNavHeader` everywhere, how about something that better suggests what the header is, i.e. that it is showing the selected scenario?

Reply:
Makes sense in the commented line, but not everywhere else, as components should be reusable and shouldn't care about content